clients/python-wrapper/lakefs/branch.py

```diff
@@ -43,7 +43,7 @@ def create(self, source_reference_id: str | Reference, exist_ok: bool = False) -
 
             branch = lakefs.repository("<repository_name>").branch("<branch_name>").create("<source_reference>")
 
-        :param source_reference_id: The reference to create the branch from
+        :param source_reference: The reference to create the branch from (reference ID or object)
```

Comment:
suggestion: think we should be more specific than object
```suggestion
        :param source_reference: The reference to create the branch from (reference ID, Reference object or Commit object)
```